server/metrics.go

```diff
@@ -177,6 +180,14 @@ var (
 			Help:      "Bucketed histogram of processing time (s) of handled forward tso requests.",
 			Buckets:   prometheus.ExponentialBuckets(0.0005, 2, 13),
 		})
+
+	regionRequestCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "pd",
+			Subsystem: "server",
+			Name:      "region_request_cnt",
+			Help:      "Counter of region request.",
+		}, []string{"request", "caller_id", "caller_component", "err_msg"})
```

Comment:
/Can we make it more simple, e.g., request-component-module?

Reply:
`ScanRegions-tidb-server->tidb-tikv-driver` is kind of confusing and not readable.

Reply:
request-component-module LGTM

However, there are `caller_id` and caller_component in the current code. If we use `request-component-module` here, it will cause confusion with the term `component`.

> ScanRegions-tidb-server->tidb-tikv-driver is kind of confusing and not readable.

It is indeed easy to get confused, and I am still thinking of a better format. Since we chose to use a hyphen as a connector in the component, continuing to use a hyphen in Grafana might look a bit strange.

<img width="745" alt="image" src="https://github.com/user-attachments/assets/95872bd5-2644-4fe4-8d13-b7d0283dcc36" />

Maybe we should name the component using camel case? Such as `RegionCache`

Reply:
What do you think? cc @lhy1024 @JmPotato @bufferflies

Reply:
`request-id-component-msg` is clear enough to me.